amr-wind/wind_energy/actuator/wing/wing_ops.H

```diff
@@ -141,22 +137,43 @@ struct ComputeForceOp<
             grid.pos, wdata.vel_tr, npts, time.current_time(), time.new_time(),
             wdata.motion_type, wdata.s_period, wdata.s_vector);
 
+        // Build the local reference frame
+        vs::Vector wspan = wdata.end - wdata.start;
+
+        // Use the global coord to orient the blade
+        // The default is for inflow in the x direction
+        auto blade_x = wdata.blade_x.unit();
+        auto blade_y = wspan.unit();
+        auto blade_z = (blade_x ^ blade_y).unit();
+        // Ensure orthogonality for new reference frame
+        blade_x = (blade_y ^ blade_z).unit();
```

Comment:
should we just `auto blade_x = (blade_y ^ blade_z).unit();` and remove lin 145? Should all these be const?

Reply:
nvm, you can substitute line 145 and make them all const